app/controllers/expenses_controller.rb

```diff
@@ -9,4 +9,19 @@ def new
     @expense = Expense.new
   end
 
+  def create
+    @expense = Expense.new(expense_params)
+
+    if @expense.save
+      redirect_to expenses_path
```

Comment:
Maybe a good place to add some flash info that creation has been completed successfully?